.circleci/config.yml

```diff
@@ -36,6 +34,10 @@ jobs:
       - run: gem --version
       - run: bundle --version
       - run: bundle install --gemfile=<< parameters.gemfile >>
+      - run: |
+          if [ "<< parameters.docker-image >>" != "ruby:2.2" ]; then
```

Comment:
If it works I think it might be slightly more expressive to use `> "ruby:2.2"` instead of `!= "ruby:2.2"`.

Reply:
Addressed in https://github.com/freerange/mocha/pull/741/commits/c37c574b0ec13cab77e19367e0bcedb54a531b92